pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -67,6 +67,20 @@ var OSDRequiredRoles = []string{
 	"roles/compute.admin",
 }
 
+// OSDSREConsoleAccessRoles is a list of Roles that a service account
+// required to get console access.
+var OSDSREConsoleAccessRoles = []string{
+	"roles/compute.admin",
+	//"roles/iam.organizationRoleAdmin",
+	"roles/editor",
+	//"roles/resourcemanager.organizationViewer",
```

Comment:
If these aren't needed, we should delete them.

Reply:
They are needed, but seems `roles/resourcemanager.organizationViewer` is deprecated and `roles/iam.organizationRoleAdmin` cannot be created using the `SetIAM` function. @sam-nguyen7

Reply:
that means this code is still WIP? Or you want to merge it like this for now but we know it won't work for future changes?

Reply:
This part is blocked https://coreos.slack.com/archives/CMD1L32M9/p1600780878019500. I think it is okay to ignore this first. WDYT? I can add a TODO.

Reply:
I will be reaching out to the google TAM to follow up on this. @yeya24 you can remove these roles for this PR. Thank you!